simulate-event.js

```diff
@@ -340,7 +339,7 @@ module.exports = function (element, type, options) {
   options = extend({
     bubbles: true,
     cancelable: true
-  }, eventOptions[eventType](element, type, options), options)
+  }, eventOptions[eventType](type, options), options)
```

Comment:
Those are only needed to provide the positional arguments for the old `init*` functions, from my understanding.